client/src/ingredients/browser-compatibility-table/browser-support-notes.js

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { BrowserSupportNote } from "./browser-support-note";
+
+function shouldDisplayBlock(blockDisplayed, versionAdded) {
+  if (blockDisplayed) {
+    return !!versionAdded;
+  }
+
+  return true;
+}
+
+export function BrowserSupportNotes({ indexNote, blockElementTag, noteElementTag, noBlocks }) {
+  let blockDisplayed = false;
+  let browserSupportNotes = [];
+  let currentNoteContent, currentNote;
+
+  if (Array.isArray(indexNote.support) && indexNote.notes.length === 0) {
+    browserSupportNotes.push(
+      <BrowserSupportNote
+        key={`note-empty-${indexNote.index}`}
+        indexNote={indexNote}
+        versionAdded={indexNote.support[0].version_added}
+        noteContent={currentNoteContent}
+        noteType="none"
+        blockElementTag={blockElementTag}
+        noteElementTag={noteElementTag}
+        displayBlock={!noBlocks && shouldDisplayBlock(blockDisplayed, indexNote.support[0].version_added)}
+        displayNote={false}
+      />
+    );
+    blockDisplayed = true;
+  } else if (indexNote.notes.length > 0) {
+    browserSupportNotes.push(
+      indexNote.notes.map((note, index) => {
+        currentNoteContent = `<abbr class="only-icon" title="See implementation notes"><span>Notes</span><i class="ic-footnote"></i></abbr> ${note}`;
```

Comment:
I can't really see what's going on or if this is just a matter of prototype-stage, but why isn't this:
```jsx
const currentNoteContent = <><abbr className="only-icon" title="See implementation notes"><span>Notes</span><i className="ic-footnote"></i></abbr> {note}</>;
```
It would definitely avoid the `dangerouslySetInnerHTML` in `BrowserSupportNoteContent` and it means we don't have to worry about the `note` string, from the BCD data, being something like `This is an innocent note <script>alert(xss)</script>`